tests/core/Jalasoft.TeamUp.Projects.Core.Tests/PostProjectCoreTests.cs

```diff
@@ -47,5 +47,36 @@ public void PostProject_Return_Project()
             var result = this.projectsService.PostProject(stubProject);
             Assert.IsType<Project>(result);
         }
+
+        [Fact]
+        public void PostProject_Returns_ValidationException()
+        {
+            var stubProject = new Project()
+            {
+                Id = Guid.Parse("5a7939fd-59de-44bd-a092-f5d8434584de"),
+                Name = "Name Example",
+                Description = "Description Example",
+                Contact = new Contact()
+                {
+                    Name = "Jose Ecos",
+                    IdResume = Guid.Parse("5a7939fd-59de-44bd-a092-f5d8434584de")
+                },
+                Logo = "https://www.example.com/images/dinosaur.jpg",
+                MemberList = new Contact[1]
+                {
+                    new Contact
+                    {
+                        Name = "Paola Quintanilla",
+                        IdResume = new Guid("536316e6-f8f6-41ea-b1ce-455b92be9303")
+                    }
+                },
+                State = true,
+                TextInvitation = "You are invited to be part of TeamUp",
+                CreationDate = DateTime.Today.AddDays(-10)
+            };
+            this.mockRepository.Setup(repository => repository.Add(stubProject)).Throws(new FluentValidation.ValidationException("BadRequest"));
```

Comment:
this exception should be thrown in the service not in the DAL the repository only returns internal server error, so you should get the exception before even reaching the dal, for this you need to send a Project that will fail the validation, if you return the exception from the DAL we are not testing what we actually need to test

Reply:
Done